osu.Game/Overlays/BeatmapSet/Scores/ScoreTable.cs

```diff
@@ -65,11 +65,13 @@ public IReadOnlyList<ScoreInfo> Scores
                 for (int i = 0; i < value.Count; i++)
                     backgroundFlow.Add(new ScoreTableRowBackground(i, value[i], row_height));
 
-                Columns = createHeaders(value[0]);
+                Columns = createHeaders(value.First());
                 Content = value.Select((s, i) => createContent(i, s)).ToArray().ToRectangular();
             }
         }
 
+        public bool IsBeatmapRanked { get; set; }
```

Comment:
I'd rename to
```suggestion
        public bool ShowPerformanceColumn { get; set; }
```
It gets the point across more directly and has the nice effect of explicitly expressing at usage site that performance points are only shown for ranked maps.